charts/vald/templates/index/job/readreplica/rotate/clusterrole.yaml

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
Could you please also add `NetworkPolicy` resource?? 🙏

Reply:
Thank you for pointing out.
[6544457](https://github.com/vdaas/vald/pull/2242/commits/654445786e83125938ca7b5849860263bda0fce0)